lib/topics/object_model.rb

```diff
@@ -2,4 +2,24 @@
 
 # exercise object model
 module ObjectModel
+  # Define Linux Friendly module
+  module LinuxFriendly
+    def fork_process
+      super
+    end
+  end
+  # Define Desktop class
```

Comment:
empty lines are needed between logical parts. New class/module is a new logical part of an application